Script/AtomicEditor/ui/frames/menus/ProjectFrameMenu.ts

```diff
@@ -101,6 +101,11 @@ class ProjectFrameMenus extends Atomic.ScriptObject {
                 return true;
             }
 
+            if (refid == "force_reimport_folder") {
+                ToolCore.assetDatabase.reimportAllAssetsInDirectory(path);
```

Comment:
IIRC there is already a change scanner, and reimports do typically happen. We currently have an issue with certain cached objects not being updated when their dependencies are though (sorry no clean repro yet), and this is a bit of a workaround to avoid re-importing everything when that case pops up.

I'll do the rename, and include it when we PR something else. I might have per-project JS editor plugins coming in this week, so it shouldn't be long.